openfisca_core/holders.py

```diff
@@ -121,6 +121,8 @@ def compute(self, period = None, **parameters):
         column = self.column
         accept_other_period = parameters.get('accept_other_period', False)
 
+        if self.column.is_neutralized:
```

Comment:
Since this check seems to occur before each `self.get_from_cache`, should it me moved to `get_from_cache` for once?